webui/react/src/hooks/useModal/Project/useModalProjectMove.tsx

```diff
@@ -124,28 +118,24 @@ const useModalProjectMove = ({ onClose, project }: Props): ModalHooks => {
         type: ErrorType.Server,
       });
     }
-  }, [destinationWorkspaceId, project.id, project.name, workspaces]);
-
-  const getModalProps = useCallback(
-    (destinationWorkspaceId?: number): ModalFuncProps => {
-      return {
-        closable: true,
-        content: modalContent,
-        icon: null,
-        okButtonProps: { disabled: !destinationWorkspaceId },
-        okText: 'Move Project',
-        onOk: handleOk,
-        title: 'Move Project',
-      };
-    },
-    [handleOk, modalContent],
-  );
+  }, [form, project.id, project.name, workspaces]);
+
+  const getModalProps = useMemo((): ModalFuncProps => {
+    return {
+      closable: true,
+      content: modalContent,
+      icon: null,
+      okButtonProps: { form: FORM_ID, htmlType: 'submit' },
```

Comment:
fix: not sure why, but moving projects doesn't work with the enter button

Reply:
https://stackoverflow.com/questions/2968693/why-doesnt-hitting-enter-when-a-select-is-focused-submit-the-form
Enter to submit doesnt apply to `select`, so i reverted it.